README.md

```diff
@@ -407,12 +432,13 @@ convert fortify2hdf         Translate a Fortify results FVDL file into a Heimdal
 convert twistlock2hdf        Translate a Twistlock CLI output file into an Heimdall
                              Data Format JSON file
   FLAGS
-  -h, --help            Show CLI help.
-  -i, --input=<value>   (required) Input Twistlock file
-  -o, --output=<value>  (required) Output HDF file
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input Twistlock file
+    -o, --output=<value>  (required) Output HDF file
+    -w, --with-raw        Append raw input file to HDF output file
 
   EXAMPLES
-    saf convert twistlock2hdf -i twistlock.json -o output-hdf-name.json
+    saf convert twistlock2hdf -i twistlock.json -o output-hdf-name.json -w
```

Comment:
We want that to be there, but we don't want to promote it as default behavior.  Consequently, don't include `-w` in the example commands.